nodes/control/cfg01.mk20-stacklight-basic.local.yml

```diff
@@ -30,3 +30,32 @@ parameters:
     system:
       name: cfg01
       domain: mk20-stacklight-basic.local
+  salt:
+    master:
+      environment:
+        dev:
+          formula:
```

Comment:
FTR collectd and heka formulas are defined in classes/system/salt/master/formula/git/stacklight.yml